### ISSUE TITLE
Add healthcheck support to Dockerized config

### ISSUE THREAD

**Issue by m-roberts** [mockoon/mockoon#945]
I have a Compose project where I am mocking an external server, but I cannot use a healthcheck currently.

```
  mock-server:
    platform: linux/amd64
    image: mockoon/cli:latest
    ports:
      - "3000:3000"
    command: [
        "--data",
        "data",
        "--port",
        "3000",
      ]
    healthcheck:
      test:
        ["CMD-SHELL", "curl -f http://localhost:3000/ || exit 1"]
      interval: 5s
      timeout: 5s
      retries: 5
      start_period: 10s
    volumes:
      - ./docker/mockoon/mock-server.json:/data:readonly
```

`$ docker inspect --format "{{json .State.Health }}"  | jq`
```
{
  "Status": "unhealthy",
  "FailingStreak": 52,
  "Log": [
    {
      "Start": "2023-02-11T23:53:46.37921184Z",
      "End": "2023-02-11T23:53:46.444886757Z",
      "ExitCode": 1,
      "Output": "/bin/sh: curl: not found\n"
    },
    {
      "Start": "2023-02-11T23:53:51.449255343Z",
      "End": "2023-02-11T23:53:51.501159343Z",
      "ExitCode": 1,
      "Output": "/bin/sh: curl: not found\n"
    },
    {
      "Start": "2023-02-11T23:53:56.508517387Z",
      "End": "2023-02-11T23:53:56.571762262Z",
      "ExitCode": 1,
      "Output": "/bin/sh: curl: not found\n"
    },
    {
      "Start": "2023-02-11T23:54:01.577323667Z",
      "End": "2023-02-11T23:54:01.64145375Z",
      "ExitCode": 1,
      "Output": "/bin/sh: curl: not found\n"
    },
    {
      "Start": "2023-02-11T23:54:06.645946377Z",
      "End": "2023-02-11T23:54:06.703083419Z",
      "ExitCode": 1,
      "Output": "/bin/sh: curl: not found\n"
    }
  ]
}
```

This particular attempt at a healthcheck is clearly failing as `curl` is not installed. I don't want to simply advocate for adding `curl` to the image, as it might be possible to add a HEALTHCHECK to the Dockerfile another way.

For example, I have tried to use `mockoon-cli info` and `mockoon-cli list` to infer if it is actually running, but I always get "No process is running" error in this context.

I am not particularly fussed what the solution is - my criteria is that it is guaranteed to be ready to use when brought up with `docker compose up --detach --wait`.

**Comment by 255kb** [comment]
@m-roberts I don't think so. Currently, all the 4 packages are released all at once. 
I plan to work in the coming days on a new release workflow. There are two needs: synchronize the versions so there is no doubt anymore about which CLI is compatible with which desktop, and having standalone releases of the CLI. 
After that it will be possible. 
But I plan to release in the coming weeks, so the wait shouldn't be long.

**Comment by 255kb** [comment]
This has been released in [v3.0.0](https://mockoon.com/releases/3.0.0/).
